virtualizor.py

```diff
@@ -472,11 +487,34 @@ def get_install_server_info(hosts_definition):
     }
 
 
+def get_gateway_info(hosts_definition):
```

Comment:
I don't like this big cut'n past. I would prefer to see a new `profile` parameter and the function renamed to something like `get_node_by_profile`.